### PR TITLE
RFC-422: Request for approval of Data Facility ConOps

### PR DESCRIPTION
**Pull request by jdswinbank**

### CODE REVIEW

File: docs/L1_Services/LSSTCam_Prompt_Processing_Services.tex [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+deep drilling program and TBD other programs. Other observing programs
+may also include AP as a science code, or may have codes of their own.
+
+\subparagraph{Calibration Operations}
+
+In addition to collecting data for science programs, the telescope and
+camera are involved in many calibration activities.
+
+the Baseline Calibrations include flats and biases. Darks are not anticipated.
+LSST has a collumanted Beam projector calibration device as well.
```
Comment (jdswinbank) [inline docs/L1_Services/LSSTCam_Prompt_Processing_Services.tex]: DMS-REQ-0130 is "The DMS shall produce and archive Calibration Data Products... including at least... Dark correction frames". See also LDM-151 §4.2.4.

(Also, looks like there's some proofreading needed on the above paragraph.)

**Comment by timj** [comment]
@jdswinbank I think `draft` branch has been used as an integration branch rather than `master`. Maybe that explains the conflicts shown on this PR? We can't rebase draft because we have a tag on there from the version we put on docushare for the review.

**Comment by timj** [comment]
Actually, for the review I merged this branch to `draft` and made some improvements, but @mgelman2 and @dpetravick continued to work on this branch rather than `draft`. I'll have a go at fixing things up and getting a coherent `master` branch out of this. I'll change the base branch of this PR to `draft`.

**Comment by jdswinbank** [comment]
Yeah — when I looked at this it wasn't clear what was supposed to merge into what, and I just needed somewhere to make comments....

... which are now, unfortunately, on a closed PR. So what's the correct workflow? Comment on #4? That seems to contain a bunch of stuff going back 18 months, but maybe that's all relevant for baselining.

**Comment by timj** [comment]
This document is a complete rewrite of the baseline. The 18 months of history is the history that is attempting to be baselined -- this PR was some cleanup of the unbaselined version that was presented to the reviewers in the summer. The real changes are all on #4 and normal policy would be for me to merge that PR to master and then create a RFC ticket branch. One downside of the new policy is that reviewers of the RFC edition can't see any of the commits that they are commenting on on github so we can't easily use github for line by line commentary.

**Comment by timj** [comment]
Oh, and I did manage to combine the two branches: the CRLF line endings and trailing whitespace were the main problems.

**Comment by jdswinbank** [comment]
...so what does that mean in practice? Are we following "normal policy" or not?

**Comment by timj** [comment]
I intend to follow normal practice and merge to master and create the RFC ticket branch for edits relating to review and for release. I was mainly wondering if you wanted to stop me doing that if it made it easier for you to comment on #4 before merging.

**Comment by jdswinbank** [comment]
I am happy to comment wherever I'm told to comment. :-) I'll hold off for now until I see you creating a new PR.

**Comment by timj** [comment]
There might not be a PR since we don't merge RFC branches back to master as such...(they are release branches). I can create an empty PR if you want, but I think the intent is for commentary to be on the RFC itself for the record.

**Pull request merged**